.github/workflows/publish_docker.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
Could this be moved to a setting, similar to [registry](https://github.com/NeonGeckoCom/.github/blob/master/.github/workflows/publish_docker.yml#L11-L13) so we can specify non-default platforms?

Reply:
Good suggestion! Incorporated.